### DISCUSSION TITLE
Please add traditional Chinese interface ...

### DISCUSSION THREAD

**Discussion by ioc2e3** [Jelmerro/Vieb#534] [Q&A]
Please add traditional Chinese interface ...

please ...

.

**Comment by Jelmerro** [comment]
It will be a long time effort to translate Vieb into multiple languages, but at the moment it is not a priority. I would need to find native speakers of those language that are willing to translate a lot of text to their language to be able to make this work. I will hopefully work on translation support in the future, and will let you know when translators are needed.

**Comment by Jelmerro** [comment]
I at least started work on the feature, not the translations itself, here: https://github.com/Jelmerro/Vieb/tree/feature/translate
I will let you know when it is ready to receive translations, for now I'm planning to add full Dutch and English, and partial Japanese myself, but anyone is welcome to contribute more languages of course! I'll keep you posted.
